fluster/utils.py

```diff
@@ -124,6 +124,26 @@ def extract(filepath: str, output_dir: str, file: Optional[str] = None) -> None:
     else:
         raise Exception(f"Unknown tarball format {filepath}")
 
+def copy(origin_path, dest_path, filename):
```

Comment:
I don't see this function being used anywhere. Can you double-check?

Reply:
I have removed it.